src/components/Auth.vue

```diff
@@ -0,0 +1,13 @@
+<template lang="html">
+  <div id="firebaseui-auth-container"></div>
+</template>
+
+<script>
+  import { authUi, authUiConfig } from '../firebase';
+  export default {
+    name: 'auth',
+    mounted() {
+      authUi.start('#firebaseui-auth-container', authUiConfig);
```

Comment:
This initialises the AuthUI stuff somehow "authUi.start" 

The first argument identifies the id of the html element that will be modified

The second argument.... calls in some config stuff from the webpack modules?

Reply:
:+1 :+1 :+1